src/tracing.ts

```diff
@@ -27,12 +27,12 @@ export function enableTracing(service: Service, tracingMode: TracingMode, handle
       lambda: true,
     };
   }
-  if (tracingMode === TracingMode.HYBRID || tracingMode === TracingMode.DD_TRACE) {
-    handlers.forEach(({ handler }) => {
-      handler.environment ??= {};
-      const environment = handler.environment as any;
-      environment[ddTraceEnabledEnvVar] = true;
-      environment[ddMergeXrayTracesEnvVar] = tracingMode === TracingMode.HYBRID;
-    });
-  }
+  handlers.forEach(({ handler }) => {
```

Comment:
Can you help me understand why we removed the conditional check for either tracing mode? Is that somehow handled by the `environment` variable instead?

Reply:
The tracingMode was defined by the config (not at the function-level) so checking tracingMode isn't necessary because the config might not match the function-level variables

Reply:
Makes sense!